trim_name.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './name_decorator'
+
+class TrimeName < NameDecorator
+  def correct_name
+    nameable.correct_name[0, 10]
+  end
```

Comment:
First, this method should check whether the name has more than 10 characters or not if it has it should trim if not it will return the value. You can use the [ternary operator](https://syntaxdb.com/ref/ruby/ternary#:~:text=The%20ternary%20operator%20is%20used,behaving%20similar%20to%20a%20function.). In your case it should be something like this 👇 

```
nameable.correct_name.length > 10 ? nameable.correct_name[0..9] : nameable.correct_name
```